conf/cortx_rgw.conf

```diff
@@ -9,6 +9,21 @@
     motr tracing enabled = true
     rgw enable apis = s3, s3website, swift, swift_auth, admin, sts, iam, notifications
     rgw cache enabled = false
+    rgw thread pool size = 10
+    rgw max concurrent request = 20
```

Comment:
@andriytk , @sachinpunadikar  please suggest default value for 'rgw max concurrent request' parameter

Reply:
As per Andriy, below value should be 
rgw max concurrent request = 1024 (the default mentioned in RGW doc)

Reply:
Thanks for confirming, @sachitanands lets go ahead with 1024 default value

Reply:
thanks Sachin Chetan .. i have updated this 1024 as default value.